nucypher/policy/policies.py

```diff
@@ -183,7 +170,7 @@ class Policy(ABC):
     An edict by Alice, arranged with n Ursulas, to perform re-encryption for a specific Bob.
     """
 
-    POLICY_ID_LENGTH = 16
+    ID_LENGTH = POLICY_ID_LENGTH
```

Comment:
Any reason we can't just use `POLICY_ID_LENGTH` directly instead of setting a new constant?